### PR TITLE
dev ltclm paerke

### PR DESCRIPTION
**Pull request by ltclm**
* new layer ch.bafu.schutzgebiete-paerke_nationaler_bedeutung_perimeter
* new structure ch.bafu.schutzgebiete-paerke_nationaler_bedeutung

[TestURL](https://mf-geoadmin3.dev.bgdi.ch/dev_ltclm_paerke/?topic=ech&lang=de&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.bafu.schutzgebiete-paerke_nationaler_bedeutung_perimeter,ch.bafu.schutzgebiete-paerke_nationaler_bedeutung&layers_opacity=0.85,0.85&X=183000.00&Y=605000.00&zoom=1)

### CODE REVIEW

File: chsdi/models/vector/bafu.py [hunk 1]
```diff
@@ -1503,23 +1503,42 @@ class schutzgebiete_aulav_uebrige_general(Base, Vector):
...
+    status = Column('status', Text)
+    teil_nummer = Column('teil_nummer', Integer)
+    zone = Column('zone', Text)
     the_geom = Column(Geometry(geometry_type='GEOMETRY',
                                dimension=2, srid=21781))
 
 register('ch.bafu.schutzgebiete-paerke_nationaler_bedeutung', paerke_nationaler_bedeutung)
 
 
+class paerke_nationaler_bedeutung_perimeter(Base, Vector):
```
Comment (loicgasser) [inline chsdi/models/vector/bafu.py]: Good habbits from now one:
paerke_nationaler_bedeutung_perimeter
should be
paerkeNationalerBedeutungPerimeter

and factorize what you can factorize, thanks!

**Comment by ltclm** [comment]
There is a jenkins error:
```bash
======================================================================
FAIL: Failure: AssertionError (ch.bazl.luftfahrtkarten-icao)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/.venv/local/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/loader.py", line 251, in generate
    for test in g():
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/chsdi/tests/integration/test_layers.py", line 111, in test_all_htmlpopups
    for layer, feature, extended in lc.ilayersWithFeatures():
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/chsdi/tests/integration/test_layers.py", line 57, in ilayersWithFeatures
    assert (models is not None and len(models) > 0), layer
AssertionError: ch.bazl.luftfahrtkarten-icao
```

**Comment by ltclm** [comment]
jenkins test

**Comment by ltclm** [comment]
jenkins test

**Comment by ltclm** [comment]
@mariokeusen

**Comment by AFoletti** [comment]
Thats a...ehm... "right" error. The layer has a tooltip for the shop. The BOD is already tooltip=true, but the model is not in the master yet (pending PR https://github.com/geoadmin/mf-chsdi3/pull/2036)

**Comment by ltclm** [comment]
great! 
thank you @AFoletti

**Comment by loicgasser** [comment]
jenkins test

**Pull request merged**